src/main/java/docSharing/service/DocumentService.java

```diff
@@ -202,8 +203,7 @@ public void exportFile(int documentId){
 
         String filename=document.getMetadata().getTitle();
         String content=document.getContent();
-
-        String pathFile = "C:/Users/Downloads/"+filename+".txt";
+        String pathFile = "C:\\Users\\tbz19\\Downloads\\"+filename+".txt";
```

Comment:
change to:

String home = System.getProperty("user.home");
String filePath = home+"/Downloads/" + fileName + ".txt";